pkg/blueprint/disk_customizations.go

```diff
@@ -20,7 +20,7 @@ type DiskCustomization struct {
 	// Optional, the default depends on the distro and image type.
 	Type       string                   `json:"type,omitempty" toml:"type,omitempty"`
 	MinSize    uint64                   `json:"minsize,omitempty" toml:"minsize,omitempty"`
-	Partitions []PartitionCustomization `json:"partitions,omitempty" toml:"minsize,omitempty"`
+	Partitions []PartitionCustomization `json:"partitions,omitempty" toml:"partitions,omitempty"`
```

Comment:
Thanks for finding/merging this - I know this is a trivial and obvious fix. IMHO it is good to take issues like this as an opportunity to see why they slipped through and add a (regression) test, here it seems the issue is that we never tested if the marshal/unmarshal is symetric, I opened https://github.com/osbuild/blueprint/pull/19 that should help. Ideally we would have a test that has a full blueprint in TOML/json and then do the same dance as in 19 but I think this is a reasonable starting point.